### DISCUSSION TITLE
Styling Probleme und Deploy Fehler

### DISCUSSION THREAD

**Discussion by ostermann-felix** [neuefische/questions#261] [Web]
Halle Friends, 

ich kämpfe gerade ein wenig mit dem Styling. Ich würde gerne die Kategorie Icons mit der Beschreibung welche darunter steht mittig und angepasst an das Rahmenquadrat anzeigen lassen. Die Schrift ein Stück kleiner und das Icon dementsprechend größer.

![Bildschirmfoto 2022-10-13 um 11 09 20](https://user-images.githubusercontent.com/94837451/195554717-9a512b07-d3ad-4c49-ab04-0c0199d3c00c.png)

Als ich gerade auch den PR machen wollte wurden mir wieder Fehler im Deployment etc. angezeigt. Weiß da jemand eventuell das Problem? https://github.com/ostermann-felix/capstone-project/pull/10

LG und Küsschen aufs Nüsschen

**Comment by ostermann-felix** [comment]
Update: Styling fixed, Deployment nach wie vor defekt.

**Comment by MerleMak** [comment]
Hi Felix, Can you please run `npm run build` and send a screenshot of the error message?

**Reply by ostermann-felix** [reply]
![Bildschirmfoto 2022-10-13 um 12 29 46](https://user-images.githubusercontent.com/94837451/195573592-433ef266-0ed8-40c2-ae63-fe362a0c4a51.png)

Still broken after I pushed and tried again
